resources/create-resource-default-credential.js

```diff
@@ -36,64 +36,55 @@ const emailAlias = process.env["EMAIL-ALIAS"];
   
-    // Use Azure SDK for Resource Management
-    const resourceManagementClientContext = new ResourceManagementClientContext(credentials, subscriptionId);
-    const resources = new Resources(resourceManagementClientContext);
-
-    // These are specific to the Azure Cognitive Services Face API
-    const resourceProviderNamespace = "Microsoft.CognitiveServices";
-    const parentResourcePath = "";
-    const apiVersion = "2017-04-18";
-
-    let parameters = {
-      type: "Microsoft.CognitiveServices/accounts",
-      location: "eastus",
-      tags: {
-        alias: emailAlias,
-        app: appName
+  // Use Azure SDK for Resource Management
+  const client = new ResourceManagementClient(credentials, subscriptionId);
+
+  // These are specific to the Azure Cognitive Services Face API
+  const resourceProviderNamespace = "Microsoft.CognitiveServices";
+  const parentResourcePath = "";
+  const apiVersion = "2017-04-18";
+
+  let parameters = {
+    type: "Microsoft.CognitiveServices/accounts",
+    location: "eastus",
+    tags: {
+      alias: emailAlias,
+      app: appName
+    },
+    sku: {
+      name: "F0"
+    },
+    kind: "Face",
+    properties: {
+      networkAcls: {
+        defaultAction: "Allow",
+        virtualNetworkRules: [],
+        ipRules: []
       },
-      sku: {
-        name: "F0"
-      },
-      kind: "Face",
-      properties: {
-        networkAcls: {
-          defaultAction: "Allow",
-          virtualNetworkRules: [],
-          ipRules: []
-        },
-        privateEndpointConnections: [],
-        publicNetworkAccess: "Enabled"
-      }
+      privateEndpointConnections: [],
+      publicNetworkAccess: "Enabled"
     }
-  
-    // Use Date as part of the resource name convention
-    const date = new Date();
-    const createdDate = date.toJSON().slice(0, 10)
-
-    const resourceType = "accounts";
-    const resourceName = `${parameters.tags.alias}-${resourceGroupName}-${resourceType}-${parameters.sku.name}-${createdDate}`;
-
-    longRunningOperationResult = await resources.beginCreateOrUpdate(resourceGroupName, resourceProviderNamespace, parentResourcePath, resourceType, resourceName, apiVersion, parameters);
-
-    console.log(longRunningOperationResult)
-    
-  } catch (err) {
-    console.log(err);
   }
 
+  // Use Date as part of the resource name convention
+  const date = new Date();
+  const createdDate = date.toJSON().slice(0, 10)
+
+  const resourceType = "accounts";
+  const resourceName = `${parameters.tags.alias}-${resourceGroupName}-${resourceType}-${parameters.sku.name}-${createdDate}`;
+  const longRunningOperationResult = await client.resources.beginCreateOrUpdate(resourceGroupName, resourceProviderNamespace, parentResourcePath, resourceType, resourceName, apiVersion, parameters);
+  return longRunningOperationResult;
 }
 
-createAzureFaceResource().then(() => {
-  console.log("done");
-}).catch((err) => {
-  console.error(err);
-});
```

Comment:
keep line 95 to line 99

Reply:
What lint/prettify are you running across the whole repo? If it isn't in a package.json at the root with a script entry, the next time anyone works on these files, they become out of sync. Either add a package.json to represent that work or don't add linting/prettify to each file - it would just cause more work for every PR after.